src/Repository/Util.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace PolderKnowledge\EntityService\Repository;
+
+use Doctrine\Common\Collections\Criteria;
+
+class Util
```

Comment:
I'm only missing tests for these methods. Other than that it looks good!

Reply:
I would make this class final. No need to extend it in any way.

Reply:
Both done.

Reply:
The test looks very much like the implementation though.